### ISSUE TITLE
Suggestion: Extrapolating on "Sorting data" section

### ISSUE THREAD

**Issue by bwright23ou** [LibraryCarpentry/lc-open-refine#355]
### How could the content be improved?

When going through this lesson in Learner mode, I was initially confused on how to navigate to the 'Sort' command on the top of the grid. I was able to figure it out with the helpful note on the bottom of the image, but I wonder if adding an image of the interface for the first 'Sort' command might be useful. Additionally, building on the content for the [Data Cleaning for Ecologists](https://datacarpentry.github.io/OpenRefine-ecology-lesson/05-filter-exclude-sort.html) workshop, I wonder if it might be useful to extrapolate on how sorting can be useful in OR. Thanks!

### Which part of the content does your suggestion apply to?

https://librarycarpentry.github.io/lc-open-refine/06-working-with-columns.html

**Comment by jas58** [comment]
@bwright23ou Thank you for your suggestion.  May I clarify your request before creating a change?
Are you asking to combine the Faceting and Filtering (4) part of the lesson with the Working with Columns (6)?
DataCarpentries does not have a separate Working with Columns section.
@ostephens would this be a Curriculum question?

**Comment by bwright23ou** [comment]
Hi Jennifer,

Thank you for reaching out! I am suggesting adding more explanatory content to the Working with Columns episode that clarifies a) the reasons why one might want to sort their data and b) explains the different kinds of sorts one can do (numeric, text, date, etc.). I like the explanation done on the DataCarpentries OpenRefine lesson, in particular, and included it as a reference (here's a link to the specific portion of the DataCarpentries explanation of [Sorting](https://datacarpentry.github.io/OpenRefine-ecology-lesson/05-filter-exclude-sort.html#sort)).

**Comment by jas58** [comment]
Thanks for clarifying.  Would you mind creating the draft? From the lesson
page, under the title,  use the blue pencil in a box icon, "edit this page"
near the top to then propose to us the edits you have in mind?
It won't change the public page just yet,  but will help us see which words
and exactly where you want them.

**Comment by bwright23ou** [comment]
Absolutely, I just submitted a pull request. Totally open to any feedback that could make this clearer, thank you so much for your time!
